### PR TITLE
wallet-ext: accounts settings view create new account from mnemonic

### PR DESCRIPTION
**Pull request by pchrysochoidis**
https://user-images.githubusercontent.com/10210143/218893872-378b65d4-18cd-49bd-92a0-f5fdb1a57fc6.mov

closes APPS-287

### CODE REVIEW

File: apps/wallet/src/ui/app/background-client/index.ts [hunk 1]
```diff
@@ -271,6 +271,17 @@ export class BackgroundClient {
         );
     }
 
+    public async deriveNextAccount() {
+        return await lastValueFrom(
```
Comment (Jibz1) [inline apps/wallet/src/ui/app/background-client/index.ts:275]: why the await

**Review by Jibz1 — APPROVED** [review]
LGTM

**Review by mystie711 — APPROVED** [review]
love it.

File: apps/wallet/src/ui/app/background-client/index.ts (hunk 1 above)
Comment (pchrysochoidis) [reply apps/wallet/src/ui/app/background-client/index.ts:275]: yeah it's not necessary, just copied-pasted from the function above. We should refactor this when we change the way we communicate with the background service

**Pull request merged**